client/app/cloud/project/compute/infrastructure/privateNetwork/addVRack/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
add label "translation required" to your PR please :)

Reply:
how to do this?

Reply:
done, added a label